packages/valory/services/oracle_goerli/service.yaml

```diff
@@ -0,0 +1,98 @@
+name: oracle_goerli
+author: valory
+version: 0.1.0
+description: A set of agents implementing a price oracle
+aea_version: '>=1.0.0, <2.0.0'
+license: Apache-2.0
+fingerprint:
+  README.md: bafybeig4wu7inb6ibwfgged3djyc7emopcg5n7d7gwrlotne53pp3jfgq4
+fingerprint_ignore_patterns: []
+agent: valory/oracle:0.1.0:bafybeihzdnqd4ukdkgz2f4av2m2eunhspfllnykgfw7pf37bqy77y6ivii
+network: goerli
+number_of_agents: 4
+---
+setup_args:
+  args: &id001
+    setup:
+      safe_contract_address:
+      - '0x623cE6087190B53520f00609d41d79e5B71B881F'
+      oracle_contract_address:
+      - '0xA2b06beb69b5e454759976ABa7555e2639c0522f'
+    max_healthcheck: 43200
+    broadcast_to_server: true
+valory_monitoring_backend:
+  args: &id002
+    url: https://staging.oracle-server.autonolas.tech/deposit
+public_id: valory/oracle_abci:0.1.0
+type: skill
+models:
+  0:
+  - price_api:
+      args:
+        url: https://api.coingecko.com/api/v3/simple/price
+        api_id: coingecko
+        parameters:
+        - - ids
+          - bitcoin
+        - - vs_currencies
+          - usd
+        response_key: bitcoin:usd
+        headers: null
+  - randomness_api:
+      args:
+        url: https://drand.cloudflare.com/public/latest
+        api_id: cloudflare
+  - params:
+      args: *id001
+  - server_api:
+      args: *id002
+  1:
+  - price_api:
+      args:
+        url: https://ftx.com/api/markets/BTC/USD
+        api_id: ftx
+        response_key: result:last
+        headers: null
+        parameters: null
+  - randomness_api:
+      args:
+        url: https://api.drand.sh/public/latest
+        api_id: protocollabs1
+  - params:
+      args: *id001
+  - server_api:
+      args: *id002
+  2:
+  - price_api:
+      args:
+        url: https://api.coinbase.com/v2/prices/BTC-USD/buy
+        api_id: coinbase
+        response_key: data:amount
+        headers: null
+        parameters: null
+  - randomness_api:
+      args:
+        url: https://api2.drand.sh/public/latest
+        api_id: protocollabs2
+  - params:
+      args: *id001
+  - server_api:
+      args: *id002
+  3:
+  - price_api:
+      args:
+        url: https://api.binance.com/api/v3/ticker/price
+        api_id: binance
+        parameters:
+        - - symbol
+          - BTCUSDT
```

Comment:
why do we use USDT over USD? It's not the same and we use USD for the others

Reply:
![image](https://user-images.githubusercontent.com/35799987/180742018-f0f16f80-ac47-4561-b0d5-cf9294eb740f.png)

Reply:
binance dont have that pair

Reply:
https://bitcoin.stackexchange.com/questions/95445/why-does-binance-use-usdt-instead-of-usd

yes suboptimal. So what happens when it loses it's peg? It doesn't matter much for now though, but I dislike this because it's incorrect, rather would compare all to USDT or USDC then. Since this is only minor and not PR related, I will approve of the PR.

Reply:
we already use this pair for all of our oracles, 
I have created an issue for this.
https://github.com/valory-xyz/open-autonomy/issues/1114